src/routes/safe/components/Settings/ManageOwners/RemoveOwnerModal/screens/Review/index.tsx

```diff
@@ -186,17 +185,17 @@ export const ReviewRemoveOwnerModal = ({
                 </Row>
                 <Hairline />
                 {owners?.map(
-                  (owner) =>
-                    owner.address !== ownerAddress && (
-                      <React.Fragment key={owner.address}>
+                  (safeOwner) =>
+                    sameAddress(safeOwner.address, owner.address) && (
```

Comment:
Wasn't the condition !== before? Shouldn't that be !sameAddress()

Reply:
fixed